docs/dependencies.md

```diff
@@ -5,37 +5,11 @@ reflected in the configuration file under `.github/dependabot.yml` and must be k
 
 ## Overview of Framework Versions
 
-|         | Monorepo | Sample Integrations    |
-| ------- | -------- | ---------------------- |
-| Angular | 17.3.0   | 17.1.1                 |
-| React   | 18.2.0   | 18.2.0                 |
-| Next.js | 14.1.3   | 13.4.19 (React 18.2.0) |
-
-## Playwright
-
-Skipped `v1.43.0` & `v1.44.0` for now because ::before elements in dark mode are not visible on VRT screenshots.
-
-In case it gets updated, the Porsche Design System / Playwright Docker image needs to be updated too (this might affect
-all running workflows even outside the changed branch, so this must be aligned with the other developers).
-
-1. Open `./docker/Dockerfile` and adjust the Playwright docker image with the updated npm `@playwright/test` version,
```

Comment:
Why have you removed the instructions on how to update the docker image / playwright?